src/test/java/com/uber/cadence/internal/sync/DeterministicRunnerTest.java

```diff
@@ -17,35 +17,20 @@
 
 package com.uber.cadence.internal.sync;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
Stars again :)

Reply:
Noo! I really thought i fixed it

Reply:
Ok, i think i really found it this time. Settings/preferences/editor/general/autoimports. There's an optimize imports checkbox